pkg/rasactl/status.go

```diff
@@ -20,33 +20,71 @@ import (
 
 	"github.com/RasaHQ/rasactl/pkg/status"
 	"github.com/RasaHQ/rasactl/pkg/types"
+	"helm.sh/helm/v3/pkg/release"
 )
 
-// Status prints status for a given deployment.
-func (r *RasaCtl) Status() error {
-	var d = [][]string{}
+type StatusProject string
+
+const (
+	StatusStopped    StatusProject = "Stopped"
```

Comment:
```suggestion
	StatusStopped   = "Stopped"
```

You don't have to defina a new type.